### DISCUSSION TITLE
Bref/Application class not found

### DISCUSSION THREAD

**Discussion by maverick108** [brefphp/bref#1310] [Support]
**Description:**
I am trying to run a SLIMMVC API with brief

The below code is throwing a class not found an error 
$app = new \Bref\Application;

**How to reproduce:**

composer install bref/bref or mnapoli/bref and slim 3.x
`<?php

use Bref\Bridge\Slim\SlimAdapter;

require __DIR__.'/vendor/autoload.php';

$slim = new Slim\App;
$slim->get('/dev', function ($request, $response) {
    $response->getBody()->write('Hello world!');
    return $response;
});

$app = new \Bref\Application;
$app->httpHandler(new SlimAdapter($slim));
$app->run();

?>`

Run the index.php file, you will get the error

Is there a bridge for bref and SLIM framework?

**Comment by mnapoli** [comment]
Hi, it seems you are following an outdated tutorial. I would recommend to uninstall Bref and start from scratch here: https://bref.sh/docs/installation.html

**Comment by maverick108** [comment]
I have followed those as well. But I was not able to make the SLIM framework with bref. It not routing properly. It says page not found.
Do you have any sample code for SLIM MVC API or bridge like you have for Laravel?

**Reply by mnapoli** [reply]
I don't have a pre-written example for Slim unfortunately.

Could you share the `serverless.yml` file, and maybe the `index.php` (or however the entrypoint of the Slim application is named`?

**Reply by maverick108** [reply]
A simple /hello using routes.php will help.

I was try run the SLIM website first using the below example before I move to a more complex MVC API example.
https://medium.com/@jonbaldie/setting-up-a-serverless-php-website-with-aws-lambda-and-bref-e1f9fb48d3cd

**Reply by mnapoli** [reply]
This example is outdated. As I mentioned above, start again following the official Bref documentation.

I cannot help if you don't help me help you: you didn't share the files I mentioned.
